packages/mc-scripts/src/utils/get-config-diff.js

```diff
@@ -0,0 +1,270 @@
+const chalk = require('chalk');
+
+// Since not all terminal supports colors, to make things more consistent for testing purposes,
+// during tests the color used is appended before the string instead of coloring it.
```

Comment:
Perhaps this comment is now outdated since we don't just append something before the strings but we wrap them.

Maybe we could update it to something like this:
```suggestion
// We use `chalk` to help users identify the changes we display in the terminal by using
// different colors for added/removed/updated values.
// There is a problem though since CI terminal does not support colors and that makes tests
// fail when we validate the output.
// To overcome that situation, we use a different approach in testing environment where we wrap
// relevant changes with custom tags so we can have consistent expectations
// during tests the color used is appended before the string instead of coloring it.
```

nit: we could even have a third color to distinguish between deletions and updates

Reply:
Yes, thanks. We needed to update the comment.

Reply:
For updates, red signifies what was removed and green for what was added. 
<img width="472" alt="image" src="https://user-images.githubusercontent.com/6726467/166901468-36fd1411-430d-4aa9-be2e-3f72263b0bc0.png">


I don't know if I understand you correctly, though.

Reply:
Oh! sorry. My bad.
Forget about it.